game/hud/src/components/MOTD.tsx

```diff
@@ -26,55 +27,152 @@ export interface WelcomeStyles {
-  -webkit-user-select: none;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  height: 450px;
-  background-color: rgba(0, 0, 0, 0.8);
-  border: 1px solid ${utils.lightenColor('#202020', 30)};
+  width: 800px;
+  height: 400px;
+  padding: 0px;
+  margin:0 auto;
+  background-color: gray;
+  color: white;
+  background: url("images/motd/motd-bg-grey.png") no-repeat;
+  z-index: 1;
+  border: 1px solid #6e6c6c;
+  box-shadow: 0 0 30px 0 #000;
 `;
 
-const Header = styled('div')`
-  width: 100%;
-  padding: 5px 0;
+const MOTDTitle = styled('div')`
   text-align: center;
-  color: white;
-  background-color: #202020;
-  border-bottom: 1px solid ${utils.lightenColor('#202020', 30)};
+  background: url("images/motd/motd-top-title.png") center top no-repeat;
+  margin: 0 auto -9px auto;
+  position: relative;
+  z-index: 999;
+  width: 319px;
+  height: 23px;
+  h6 {
+    color: #848484;
+    font-size: 10px;
+    text-transform: uppercase;
+    padding: 7px 0 0 0;
+    margin: 0 0 0 0;
+    font-family: 'Caudex', serif;
+  }
 `;
 
-const Content = styled('div')`
-  flex: 1;
-  color: white;
-  padding: 5px;
-  overflow: auto;
+const MOTDCorner = styled('div')`
+  position: absolute;
+  min-width: 800px;
+  min-height: 400px;
+  background:
+  url("images/motd/motd-ornament-top-left.png") left 0 top 0 no-repeat,
+  url("images/motd/motd-ornament-top-right.png") right 0 top 0 no-repeat,
+  url("images/motd/motd-ornament-bottom-left.png") left 0 bottom 0 no-repeat,
+  url("images/motd/motd-ornament-bottom-right.png") right 0 bottom 0 no-repeat;
+  z-index: 1;
+`;
+const MOTDMotdTitle = styled('div')`
+  height: 40px;
+  h4 {
+    color: #cebd9d;
+    line-height: 40px;
+    margin-left: 20px;
+  }
+`;
+const MOTDContent = styled('div')`
+  height: 285px;
+  margin-top: 0px;
+  max-height: 295px;
+  padding: 10px 20px;
+  box-sizing: border-box;
+  overflow-x: hidden;
+  overflow-y: auto;
+  border-top: 1px solid #3b3634;
+  background: rgba(0, 0, 0, 0.6);
+  z-index: 10;
+  width: calc(100% - 40px);
+  position: absolute;
 `;
 
-const Footer = styled('div')`
-  padding: 5px 0;
-  background-color: #202020;
-  text-align: center;
-  border-top: 1px solid ${utils.lightenColor('#202020', 30)};
+const MOTDFooter = styled('div')`
+  position: absolute;
+  min-width: 800px;
+  height: 55px;
+  bottom: 0;
+  left: 0;
+  background: rgba(55, 52, 51, 0.3);
+  border-top: 1px solid #3b3634;
+  z-index: 11;
 `;
 
-const DismissButton = styled('a')`
-  cursor: pointer;
+const MOTDButton = styled('div')`
+  &.btn {
+    background: url("images/motd/button-off.png") no-repeat;
+    width: 95px;
+    height: 30px;;
+    border: none;
+    margin: 12px 16px 0 16px;
+    cursor: pointer;
+    color: #848484;
+    font-family: 'Caudex', serif;
+    font-size: 10px;
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 30px;
+    &:hover {
+      background: url("images/motd/button-on.png") no-repeat;
```

Comment:
Not a problem just letting you know - you don't actually need to put quotes around urls used in styled components